client/signup/config/step-components.js

```diff
@@ -71,6 +71,7 @@ const stepNameToModuleName = {
 	'plans-ecommerce-monthly': 'plans',
 	'plans-personal-monthly': 'plans',
 	'plans-premium-monthly': 'plans',
+	design: 'design-picker',
```

Comment:
Note to self: the `design` step, when rendered, uses the component defined in `signup/steps/design-picker/`